internal/oauthserver/oauth_server.go

```diff
@@ -67,30 +63,31 @@ func NewOAuthServer(
 	oauthClient auth.OAuthClient,
 	sessionStore sessions.Store,
 	directory identity.Directory,
-) *OAuthServer {
-	storage := newStore()
+) (*OAuthServer, error) {
+	secret := []byte("my super secret signing password")
```

Comment:
this is for signing the tokens right? we need to do something better in the future?